styles.css

```diff
@@ -52,25 +54,98 @@ li {
   padding-bottom: 15%;
 }
 
-li a {
-  color: inherit;
-  text-decoration: none;
+.title {
+  cursor: pointer;
+  color: #28352f;
+  font-weight: 600;
 }
 
-.toolbar {
+/* Properties for Nav-bar */
+.navbar {
   display: flex;
+  top: 0;
+  height: 10vh;
   justify-content: space-between;
-  flex: 1;
+  align-items: center;
+  padding: 0 20px;
 }
 
-.toolbar a {
-  text-decoration: none;
+.list-item {
+  padding: 10px;
+  list-style: none;
+}
+
+.nav-link {
   color: inherit;
+  text-decoration: none;
 }
 
-#menu-icon {
-  font-size: 20px;
-  padding-right: 30px;
+/* Properties for Hamburger */
+.ham {
+  /* display:none;   */
+  padding: 5px;
+  cursor: pointer;
+  position: absolute;
+  z-index: 2;
+  right: 8%;
+}
+
+.bar1,
+.bar2,
+.bar3 {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: #101010;
+  transition: 0.4s;
+}
+
+.nav-sub {
+  position: absolute;
+  right: 0;
+  flex-direction: column;
+  flex: 0;
+  background-color: #fff;
+  color: #3a4a42;
+  border-radius: 0 0 30px 30px;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  transition: transform 0.3s ease-in;
+  transform: translate(100%);
+  top: 0;
+  height: 100%;
+  width: 100%;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  padding: 20% 0 0 10%;
+}
```

Comment:
- [ ] You did a good job on this project 👏. However, there is one issue that needs to be addressed in order for our mobile menu to resemble the original. We can see a small portion of the background after we open the menu at the top of the close icon. Please fix this because we want to make sure our design is as close to the original as possible. 👌😊

One way I have found to correct that is to increase the `height` of the `navbar` at line `64` to `15vh`.

### Our design
![Screenshot from 2023-03-07 18-32-38](https://user-images.githubusercontent.com/89228743/223517304-ec6f41b9-c093-43df-aa08-0992dc51603f.png)

### The original design
![Screenshot from 2023-03-07 18-36-19](https://user-images.githubusercontent.com/89228743/223518002-61942670-b4c9-488a-99d1-4f1863dea75f.png)